### ISSUE TITLE
can't figure it out

### ISSUE THREAD

**Issue by b9bigbill** [pathartl/BleemSync#102]
I can't figure this out I did everything im suppose to but when plug the usb in the light doesn't blink or work

**Comment by b9bigbill** [comment]
i tried 2 different usb drives

**Comment by Gab-75** [comment]
As I wrote in my previous post "the PS Classic USB port host has a current limiter capping to 100 mA" and this may cause issues with many USB devices... in this case a powered hub (with a dedicated AC adapter) can be the solution.

**Comment by FyreAnt** [comment]
I didn’t run into any of those issues, though I am running a 128gb Sandisk Extreme Plus micro SD in a small nano SD reader and it works great. I’m copying 183 games to it currently and will be testing it out shortly.

**Comment by pathartl** [comment]
Make sure the volume name is SONY and the 028c18a9-ec4b-4632-b2cf-d4e20f252e8f exists. Also make sure the light on the front blinks.

**Comment by Mystix27** [comment]
Same Probleme here, i have an ac adaptater for my ps and tested 3 differents key. I don't have ant pref file in system even if bloomSync run successfully. When i launch the ps it's like there is nothing.

**Comment by AlexanderPlaza** [comment]
You can find a list of compatible/tested USB sticks listed here:
https://docs.google.com/spreadsheets/d/1saluXC8jfWhOVT9-JXj1Kd62hL2WGtCX6zErY6AElco/

Please note that this isn't 100% and it appears to be still a work in progress. But hopefully it helps you figure out which USB Key works for you.

**Comment by kurokobo** [comment]
In my experience, the USB flash drive have to be initialized with MBR instead of GPT.
I am pleased if this helps.

**Comment by daarcm** [comment]
It most probably has to do with the compatibility of the flash controllers. I have tried two different drive enclosures, the "newer" (5 years old) 3.0 one doesn't work, although an externally powered 2.0 works. It is also likely that the newer flash controllers require additional power, although they are supposed to be 2.0 compatible, it is mentioned that the port is power limited to 100mA and the 2.0 standard is 500mA. I also purchased a 3.1 flash stick that didn't work, but older flash sticks do.
